### PR TITLE
Update to 2024.3.5

### PR DESCRIPTION
**Pull request by cddjr**

### CODE REVIEW

**Review by liaralats — COMMENTED** [review]
Update to 2024.3.5

**Review by OsOmE1 — APPROVED** [review]

**Review by dbjejcjallwl — COMMENTED** [review]
gfj

**Comment by dbjejcjallwl** [comment]
62f80cc82f4710e4b945585225dc40d8b1b99c29

File: appdata/il2cpp-functions.h
Comment (dbjejcjallwl) [inline appdata/il2cpp-functions.h:1]: 62f80cc82f4710e4b945585225dc40d8b1b99c29

**Comment by dbjejcjallwl** [comment]
> Update to 2024.3.5 

da7a3efb5ed590bd95dcea010ca8faac80231816

**Pull request merged**